contracts/Tokens/VAI/VAIController.sol

```diff
@@ -387,6 +394,38 @@ contract VAIController is VAIControllerStorageG2, VAIControllerErrorReporter, Ex
         return uint(Error.NO_ERROR);
     }
 
+    /**
+     * @notice Set the prime token contract address
+     * @param prime_ The new address of the prime token contract
+     * @return uint 0=success, otherwise a failure (see ErrorReporter.sol for details)
+     */
+    function _setPrimeToken(address prime_) external returns (uint) {
+        if (msg.sender != admin) {
+            return fail(Error.UNAUTHORIZED, FailureInfo.SET_COMPTROLLER_OWNER_CHECK);
+        }
```

Comment:
We could use the modifier `onlyAdmin` instead, right? As we are doing at `setAccessControl`. This way, we wouldn't need to return any value